webpack/scenes/Organizations/OrganizationActions.js

```diff
@@ -49,6 +49,9 @@ export const saveOrganization = (extendedParams = {}) => (dispatch) => {
         type: SAVE_ORGANIZATION_SUCCESS,
         response: data,
       });
+
+      // TODO: Necessary because of https://projects.theforeman.org/issues/26420
+      dispatch(loadOrganization());
```

Comment:
this would execute two API requests (PUT and GET) when saving organization. 
wouldn't it be more reasonable fixing the API itself, and get the requested data in the first API ?

Reply:
Yes, it would.  However, this is for an upcoming release and I don't have the expertise (or time) to get this into foreman.